lib/Inspection/VPack.h

```diff
@@ -42,6 +42,13 @@ void serialize(Builder& builder, T& value) {
   }
 }
 
+template<class T>
+[[nodiscard]] auto serialize(T& value) -> SharedSlice {
+  auto builder = Builder();
+  serialize(builder, value);
+  return builder.sharedSlice();
```

Comment:
This will copy the buffer into a new SharedSlice. To prevent the copy you could do:
```suggestion
  return std::move(builder).sharedSlice();
```
But since I could only find one usage of this serialize function in `inspection_formatter::format` I am wondering why we don't just create a local Builder instance there and use the existing serialize function that takes the builder

Reply:
Because I imagine using this `serialize` function in other places and not just in the formatter ;)